routers/extreme_netiron.php

```diff
@@ -162,6 +162,11 @@ protected function build_traceroute($parameter) {
 
     return array($cmd);
   }
+
```

Comment:
Not required if the IPv6 specific AS path regexp function is removed and its logic included in the existing AS path function.